stock_request/models/stock_move.py

```diff
@@ -69,7 +69,7 @@ def copy_data(self, default=None):
 
     def _action_cancel(self):
         res = super()._action_cancel()
-        self.mapped("allocation_ids.stock_request_id").check_done()
+        self.mapped("allocation_ids.stock_request_id").check_cancel()
```

Comment:
@kittiu Could we have your advice here as this is you who introduced this during 14.0 migration ? Thanks